tests/viz/test_exporter.py

```diff
@@ -136,25 +137,28 @@ def subdomain(request: pytest.FixtureRequest) -> SingleSubdomain:
 def test_single_subdomains(setup: ExporterTestSetup, subdomain: SingleSubdomain):
```

Comment:
I somewhat do not like that you simply changed all tests. Since the exporter tests use hardcoded vtk files, and you changed both tests and reference solutions. The updated tests do not verify that the new code does not change the previous functionality. I would have somehow liked to see first that the old tests run through (tested by github actions), and then the updated tests are added in a later commit-push, invoking a new run of the tests. Then I would trust the tests better. Sound somewhat silly to do a reverse here, but still I would be happy to get that confirmed once.

Reply:
For me it would be OK, if you open a new PR this time, just adding the changes to all core but not the tests. Then the old tests can be verified. Would this be allowed @keileg ?

Reply:
In addition, I now see that the tests for the exporter actually fail. Since it is so easy to trick the Exporter tests by simply changing the vtk files accordingly, I would even more advocate to see a green checkmark before changing the test suite.

Reply:
okay, I see your point and I try to modify the pr in this way. I would just wait the confirmation from @keileg that this is the way to proceed. I've modified all the old vtk files since now by default we get also the node ids (as the cells ids) exported.

Reply:
> For me it would be OK, if you open a new PR this time, just adding the changes to all core but not the tests. Then the old tests can be verified. Would this be allowed @keileg ?

Yes, that sounds like the right approach.

Reply:
The code is finalized. PR #1198 serves as the auxiliary PR testing the updated core with the old test suite - the tests go through. For me this PR is good to go and ready for merge. Any other opinion @keileg ? @alessiofumagalli  gave us the permission to proceed with the PR.